components/discover/defiTable.tsx

```diff
@@ -430,12 +428,12 @@ const DataTable: FunctionComponent<DataTableProps> = ({ data, loading }) => {
         <div className="flex items-center justify-center space-x-2 pt-4">
           <div className="text-sm text-muted-foreground flex gap-8">
             <div
-              className="flex"
+              className="flex modified-cursor-pointer"
               onClick={() =>
                 table.getCanPreviousPage() ? table.previousPage() : null
               }
               style={{
-                cursor: table.getCanPreviousPage() ? "pointer" : "not-allowed",
+                cursor: table.getCanPreviousPage() ? "inherit" : "not-allowed",
```

Comment:
![image](https://github.com/user-attachments/assets/d57d0176-21c9-4464-83ec-3bbcf6a85b06)
Here the cursor should be the custom "pointer" when allowed (same for both arrows). I am seing the custom default pointer (not the one for clickable elements)